src/App.js

```diff
@@ -28,12 +29,15 @@ function App() {
           <Route path="/add">
             <AddConference />
           </Route>
+          <Route path="/users">
+            <RegistrationForm />
```

Comment:
Does `/users` signify a registration? Perhaps `/user/regiser`, initally `/users` suggest to me you want to list other users